packages/tupaia-web/src/features/EnlargedDashboardItem/EnlargedDashboardItem.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React, { useReducer } from 'react';
+import styled from 'styled-components';
+import { useSearchParams } from 'react-router-dom';
+import { ViewConfig } from '@tupaia/types';
+import { URL_SEARCH_PARAMS } from '../../constants';
+import { Modal } from '../../components';
+import { Entity } from '../../types';
+import { ExportDashboardItem } from './ExportDashboardItem';
+import { EnlargedDashboardVisual } from './EnlargedDashboardVisual';
+import {
+  EXPORT_FORMATS,
+  ExportContext,
+  ExportDispatchContext,
+  exportReducer,
+  useEnlargedDashboardItem,
+} from './utils';
+import { ExportButton } from './ExportButton';
+
+const Wrapper = styled.div<{
+  $hasBigData?: boolean;
+  $applyWidth?: boolean;
+}>`
+  max-width: 100%;
+  min-width: ${({ $hasBigData, $applyWidth }) => ($applyWidth && $hasBigData ? '90vw' : 'auto')};
+  width: ${({ $hasBigData, $applyWidth }) => {
+    if (!$applyWidth) return 'auto';
+    return $hasBigData ? '90%' : '48rem';
+  }};
```

Comment:
```suggestion
  min-width: ${({ $hasBigData }) => ($hasBigData ? '90vw' : 'auto')};
  width: ${({ $hasBigData }) => {
    if (!$applyWidth) return 'auto';
    return $hasBigData ? '90%' : '48rem';
  }};
```
This can be simplified a bit now right? And is there any way of getting rid of applyWidth altogether?

Reply:
We can, but we would have to have the data download modal also as 48rem, is that okay?

Reply:
I vote yes

Reply:
Done